hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/MiniRouterDFSCluster.java

```diff
@@ -1202,4 +1202,8 @@ public void waitClusterUp() throws IOException {
       throw new IOException("Cannot wait for the namenodes", e);
     }
   }
+
+  public long getCacheFlushInterval() {
```

Comment:
Add a javadoc.
Ideally I would like to have the unit in there too.

Reply:
done